.env.example

```
@@ -1,22 +1,28 @@
-OPENROUTER_DEFAULT_MODEL="anthropic/claude-3.5-sonnet"
-OPENROUTER_FAST_MODEL="google/gemini-flash-2.5"
+OPENROUTER_STRUCTURED_MODEL="anthropic/claude-3.5-sonnet"
+OPENROUTER_FAST_STRUCTURED_MODEL="google/gemini-1.5-flash"
+OPENROUTER_THINKING_MODEL="google/gemini-1.5-flash"
 
 # --- Cerebras Settings (新規追加) ---
 CEREBRAS_API_KEY="YOUR_CEREBRAS_API_KEY"
-CEREBRAS_DEFAULT_MODEL="gpt-oss-120b"
-CEREBRAS_FAST_MODEL="llama-4-maverick-17b-128e-instruct"
+# BTLM-3B-8K-chat のようなJSONモード対応モデルを指定
+CEREBRAS_STRUCTURED_MODEL="btlm-3b-8k-chat"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)